scripts/check-dockerfiles.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
I would prefer to have only one Dockerfile (rhel based) and then script to generate CentOS based one if developer needs it. But I know that this is the tested way which is used in vmaas and engine.